### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by Varun-28**
Done with Playlist feature

- create playlist
- add to the playlist
- remove from playlist
- delete playlist

Screenshot 2022-05-21 at 15-20-35 CraftTube
Screenshot 2022-05-21 at 15-20-54 CraftTube

### CODE REVIEW

**Review by shilpe26 — APPROVED** [review]
▪️Woking as expected
▪️Code looks prettier and clean
▪️ Good to merge

File: src/pages/playlistDetail/PlaylistModal.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import { usePlaylistServerCall } from "../../context/playlistContext/usePlaylistServerCall";
+import "./playlistModal.css";
+
+function PlayListModal({ modal }) {
+  const { playlistState, setModal } = usePlaylist();
+  const { createPlaylist, addToPlaylist } = usePlaylistServerCall();
+  const [title, setTitle] = useState("");
+
+  const createClickHandler = () => {
+    if (title !== "") {
```
Comment (Hamdrive) [inline src/pages/playlistDetail/PlaylistModal.jsx:12]: you can directly pass title, as empty string will give falsy and wont pass the condition

**Review by stutisk — APPROVED** [review]
clean code 
can be merged

**Review by Hamdrive — COMMENTED** [review]
Good work done implementing playlist feature
Logic looks sound and works as expected

File: src/pages/playlistDetail/PlaylistModal.jsx (hunk 1 above)
Comment (Varun-28) [reply src/pages/playlistDetail/PlaylistModal.jsx:12]: Yes, Sure

**Review by GeetaPogul — COMMENTED** [review]
code looks good

**Pull request merged**